src/examples/candid-health/sync-candid-tasks.ts

```diff
@@ -0,0 +1,135 @@
+import { BotEvent, MedplumClient, createReference } from '@medplum/core';
+import fetch from 'node-fetch';
+
+const CANDID_API_URL = 'https://api-staging.joincandidhealth.com/api/v1/';
+
+export async function handler(medplum: MedplumClient, event: BotEvent): Promise<any> {
+  const status = 'open';
+  // Craft the queries all status of requested typs
+  const candidTaskQuery = {
+    skip: 0,
+    limit: 100,
+    status: status,
+    updated_since: '2021-01-01T00:00:00.000Z',
+    sort: 'updated_at:desc',
+  };
+
+  const result = await getCandidTasks(
+    candidTaskQuery,
+    event.secrets['CANDID_API_KEY'].valueString as string,
+    event.secrets['CANDID_API_SECRET'].valueString as string
+  );
+
+  console.log('Received Response from Candid:\n', JSON.stringify(result, null, 2));
+
+  const candidTaskResult = (await result.json()) as CandidTask[];
+
+  for (const candidTask of candidTaskResult) {
+    const encounter = await medplum.createResourceIfNoneExist(
+      {
+        resourceType: 'Encounter',
+        identifier: [
+          {
+            system: 'https://api-staging.joincandidhealth.com/api/v1/encounters',
```

Comment:
In this bot, we're using a mix of `system` strings `https://api-staging.joincandidhealth.com/api/v1` vs. `http://joincandidhealth.com`. 

I think it would be better just to stick to the later